composer.json

```diff
@@ -19,6 +19,7 @@
     "illuminate/database": "^5.8|^6.0|^7.0|^8.0"
   },
   "require-dev": {
+    "ext-pdo_sqlite": "*",
```

Comment:
why we need this change?

Reply:
The config in DatabaseTestcase.php uses a Sqlite database, so there is a dependency. Listing required extensions here makes composer give a nice message instead of getting errors while running phpunit.